content/en/graphing/using_graphs/_index.md

```diff
@@ -1,74 +1,73 @@
 
-The graphing editor has the following tabs:
+On dashboards, open the graphing editor by clicking on the pencil icon in the upper right corner. The graphing editor has the following tabs:
 
 * **Share**: Embed the graph on any external web page.
 * **JSON**: The more flexible editor which requires knowledge of the graph definition language.
-* **Edit**: The default, GUI tab for graphing options.
+* **Edit**: The default, UI tab for graphing options.
 
 When you first open the graph editor, you are on the **Edit** tab. Here you can use the UI to choose most settings. Here is an example:
 
-{{< img src="graphing/index/references-graphing-edit-window-with-y.png" alt="Graphing Edit Tab" responsive="true" style="width:75%;" >}}
+{{< img src="graphing/using_graphs/references-graphing-edit-window-with-y.png" alt="Graphing Edit Tab" responsive="true" style="width:75%;" >}}
 
 ## Configuring a graph
-To configure your graph follow this process:
+To configure your graph on dashboards, follow this process:
 
-1. [Choose the metric to graph](#choose-the-metric-to-graph)
-2. [Select the visualization](#select-your-visualization)
+1. [Select the visualization](#select-your-visualization)
+2. [Choose the metric to graph](#choose-the-metric-to-graph)
 3. [Filter](#filter)
-4. [Aggregate and Rollup](#aggregate-and-rollup)
+4. [Aggregate and rollup](#aggregate-and-rollup)
 5. [Apply additional functions](#advanced-graphing)
 6. [Title the graph](#create-a-title)
 
-### Choose the metric to graph
-
-When you create a graph, you probably have a metric in mind that you want to show. You can select that in the first dropdown under step #2, **Graph your data**. If you don't know which metric to use, you might want to start with the [Metrics Explorer][2] or a [Notebook][3]. You can also see a list of metrics in the [Metrics Summary][4].
+### Select your visualization
 
+Select your visualization from the available [widgets][2].
 
-### Select your visualization
+### Choose the metric to graph
 
-Once you select a metric to display in your graph, choose your visualization. Check the [list of all visualizations (widgets)][5].
+Choose the metric to graph by searching or selecting it from the dropdown next to **Metric**. If you don't know which metric to use, start with the [Metrics Explorer][3] or a [Notebook][4]. You can also see a list of metrics in the [Metrics Summary][5].
 
 ### Filter
 
-Once the metric and a visualization are in place, you can filter the hosts to be graphed. To the right of the metric is the **from** dropdown which defaults to *(everything)*. Click this and choose the tag key(s) to filter by. To learn more about tags, refer to the [Tagging documentation][6].
+Your chosen metric can be filtered by host or any tag using the **from** dropdown to the right of the metric. The default is `(everywhere)`. To learn more about tags, refer to the [Tagging][6] documentation.
```

Comment:
```suggestion
Your chosen metric can be filtered by host or any tag using the **from** dropdown to the right of the metric. The default is *everywhere*. To learn more about tags, refer to the [Tagging][6] documentation.
```

Using an italic here since it's not explicitly stated.

Reply:
Ok. Added the parentheses back since they display in the UI.